src/migrations/20231030091931-add-created-by-and-status-change-request-schedule.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+exports.up = function (db, cb) {
+  db.runSql(`
+        ALTER TABLE change_request_schedule
+            ADD COLUMN IF NOT EXISTS created_by integer REFERENCES users(id);
```

Comment:
Would `scheduled_by` be a better column name than `created_by` in this instance?

```suggestion
            ADD COLUMN IF NOT EXISTS scheduled_by integer REFERENCES users(id);
```

Reply:
either way is fine with me - row was created by someone is what matters

Reply:
Got it! I'd lean towards "scheduled_by", but I'll let it be up to you in the end ☺️